/pid_controller:
  ros__parameters:
    kd: 0.8
    ki: 0.01
    kp: 1.0
    goal_reach_tol: 0.05
    max_linear_velocity: 1.0
    max_angular_velocity: 2.0
    max_linear_acceleration: 0.1
    max_linear_deceleration: 0.9
    
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          durability: volatile
          history: keep_last
          reliability: reliable
    use_sim_time: false
